src/DataTable/PaginationBox.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PaginationButtons from "./PaginationButtons";
+import RowsPerPageInput from "./RowsPerPageInput";
+
+const PaginationBox = ({
+  currentPageNumber,
+  totalNumberOfPages,
+  setCurrentPageNumber,
+  rowsPerPage,
+  setRowsPerPage,
+}) => {
+  return (
+    <div className="pagination-box">
+      <PaginationButtons
+        currentPageNumber={currentPageNumber}
+        setCurrentPageNumber={setCurrentPageNumber}
+        totalNumberOfPages={totalNumberOfPages}
+      />
+      <RowsPerPageInput
+        rowsPerPage={rowsPerPage}
+        setRowsPerPage={setRowsPerPage}
+        min={1}
+        max={10}
```

Comment:
min and max can also be passed down from DataTable, but I don't want to make this refactor over complicated, so I just set them here